### PR TITLE
Add create release and publish to marketplace GitHub Actions

### PR DESCRIPTION
**Pull request by pedromcp90**
Required by #96

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
+
+      - name: Set output
+        id: vars
+        run: |
+          echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+          echo "zip_path=../doofinder-${GITHUB_REF#refs/*/}.zip" >> $GITHUB_OUTPUT
+      - name: Build
+        run: sh build.sh ${{ steps.vars.outputs.tag }}
```
Comment (albertodoofinder) [inline .github/workflows/create_release.yml]: Please, instead of using build.sh file put the instructions directly in the step

File: build.sh [hunk 2]
```diff
@@ -4,4 +4,21 @@ if [ "$#" -ne 1 ]; then
...
+rm doofinder/docker-compose.yml
+rm doofinder/composer.json
+rm -rf doofinder/.git
+rm -rf doofinder/.github
+rm doofinder/.git*
+
+zip -r doofinder-$1.zip doofinder
+mv doofinder-$1.zip ..
+cd ..
+rm -rf _build
```
Comment (ivan-doofinder) [inline build.sh]: Why are you changing this?
The `git archive` command generates the zip ready to push the release

**Review by ivan-doofinder — CHANGES_REQUESTED** [review]

File: build.sh (hunk 2 above)
Comment (pedromcp90) [reply build.sh]: Yes, but doing it that way some unwanted files are being added to the compressed file, like the `composer.json` and `docker-compose.yml`

Comment (ivan-doofinder) [reply build.sh]: Add them to the .gitattributes file so it ignores them when packaging

Comment (pedromcp90) [reply build.sh]: I've added the excluded files in the `.gitattributes` file

**Review by albertodoofinder — APPROVED** [review]

**Review by ivan-doofinder — APPROVED** [review]

**Pull request merged**